### ISSUE TITLE
documentation clarity

### ISSUE THREAD

**Issue by FractalHQ** [nanostores/nanostores#147]
Hey there!  I'm a bit confused about the `Computed` docs.

[The example](https://github.com/nanostores/nanostores#computed-stores) has a variable called `all` but doesn't explain what it is:

```ts
export const admins = computed(users, all => {
  // This callback will be called on every `users` changes
  return all.filter(user => user.isAdmin)
})
```

The typescript definition describes it as `any` without any further description, and claims it's a required argument.

My guess was that it should contain any variables I want to use inside the callback, but when I tried passing in a Map:

```ts
export const currentTheme = persistentAtom('currentTheme', 'dark', {
	encode: JSON.stringify,
	decode: JSON.parse,
})

const themes = new Map([
	['dark', { ...dark, ...brand }],
	['light', { ...light, ...brand }],
	['custom', custom],
]) satisfies ThemeMap

export const theme = computed(currentTheme, themes => themes.get(currentTheme))
```

But I get an error that says `themes2.get is not a function`, leading to even more confusion.

Perhaps I'm missing something obvious?

**Comment by FractalHQ** [comment]
I got it working rather quickly by doing:

```ts
export const theme = computed(currentTheme, () => themes.get(currentTheme.get()))
```

Although I'm still curious about the callback param.

**Comment by ai** [comment]
`all` is the value of the store.

```js
export const admins = computed(users, usersValue => {
  // This callback will be called on every `users` changes
  return usersValue.filter(user => user.isAdmin)
})
```

**Comment by ai** [comment]
I fixed docs 77b5027
